packages/create-gea/templates/mobile/src/styles.css

```
@@ -0,0 +1,136 @@
+  font-size: clamp(2rem, 5vw, 2.6rem);
+  line-height: 1.05;
+}
+
+.copy {
+  margin: 14px 0 0;
+  color: #94a3b8;
+}
+
+code {
+  color: #f8fafc;
+  font-family: 'SFMono-Regular', ui-monospace, monospace;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)